src/components/appBar/AppBar.tsx

```diff
@@ -1,34 +1,109 @@
+import { useState, useEffect, Suspense, MouseEvent } from 'react';
 import { NavLink, Outlet, useLocation } from 'react-router-dom';
-import * as React from 'react';
-import Box from '@mui/material/Box';
-import Tabs from '@mui/material/Tabs';
-import Tab from '@mui/material/Tab';
+import {
+  Container,
+  Box,
+  Tabs,
+  Tab,
+  IconButton,
+  Menu,
+  MenuItem,
+} from '@mui/material';
+import DragHandleIcon from '@mui/icons-material/DragHandle';
+
+import logo from '../../../public/logo.svg';
 
 const AppBar = () => {
   const location = useLocation();
-  const [value, setValue] = React.useState(
-    location.pathname === '/garden' ? 1 : 0
-  );
+  const [isMobile, setIsMobile] = useState(window.innerWidth <= 500);
+  const [menuAnchorEl, setMenuAnchorEl] = useState<null | HTMLElement>(null);
+
+  const handleMenuToggle = (event: MouseEvent<HTMLButtonElement>) => {
+    setMenuAnchorEl(event.currentTarget);
+  };
 
-  const handleChange = (_event: React.SyntheticEvent, newValue: number) => {
-    setValue(newValue);
+  const handleMenuClose = () => {
+    setMenuAnchorEl(null);
   };
 
+  useEffect(() => {
+    const handleResize = () => {
+      setIsMobile(window.innerWidth <= 500);
+    };
+
+    window.addEventListener('resize', handleResize);
+
+    return () => {
+      window.removeEventListener('resize', handleResize);
+    };
+  }, []);
```

Comment:
i like this solution. Smart.
at the same time you probably will use the same logic in other components to.
In my experience we usually put it in a custom hook, if you need tip on how to do it, lmk ;) 
at the same time of you wanna leave it as it is, its okay by me, we can fix it later